### PR TITLE
Adds column name in html table vectorizer

### PR DESCRIPTION
**Pull request by Shree7676**
#948
cc @jeromedockes 
@ PyData Paris Sprint

### CODE REVIEW

**Comment by Shree7676** [comment]
to fix the codecov/patch 
should I just update the test case name as test_sk_visual_block_

File: skrub/tests/test_table_vectorizer.py [hunk 1]
```diff
@@ -714,6 +714,16 @@ def test_bad_specific_cols():
         TableVectorizer(specific_transformers=[(None, [0])]).fit(None)
 
 
+def test_sk_visual_block_():
```
Comment (jeromedockes) [inline skrub/tests/test_table_vectorizer.py:717]:
```suggestion
def test_sk_visual_block():
```

File: skrub/tests/test_table_vectorizer.py [hunk 2]
```diff
@@ -714,6 +714,16 @@ def test_bad_specific_cols():
...
+    X = _get_clean_dataframe()
+    vectorizer = TableVectorizer()
```
Comment (jeromedockes) [inline skrub/tests/test_table_vectorizer.py:719]: here we could check the repr html of the unfitted estimator to check it works and cover the missing line that codecov is complaining about

```python
unfitted_repr = vectorizer._repr_html_()
assert 'TableVectorizer' in unfitted_repr
```

**Review by jeromedockes — APPROVED** [review]
thanks a lot @Shree7676 !!

**Pull request merged**